vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -39,7 +41,9 @@ cat <<EOF > vhd-publishing-info.json
     "sku_name" : "$sku_name",
     "offer_name" : "$OFFER_NAME",
     "hyperv_generation": "${HYPERV_GENERATION}",
-    "image_version": "${IMAGE_VERSION}"
+    "image_version": "${IMAGE_VERSION}",
+    "os_sku" : "${OS_SKU}",
+    "os_version" : "${OS_VERSION}"
```

Comment:
do you have an example of generated content for ubuntu? Want to make sure there is not much confusion between os_sku and sku_name, os_version etc.

Reply:
Here's an example of an Ubuntu JSON. The os_sku and and os_version mostly stay the same (except 1604), and the sku_name is still version number concatenated with feature flags.
```json
{
    "vhd_url" : "<URL>",
    "os_name" : "Linux",
    "sku_name" : "1804containerd",
    "offer_name" : "Ubuntu",
    "hyperv_generation": "V1",
    "image_version": "0.0.1",
    "os_sku" : "Ubuntu",
    "os_version" : "18.04"
}

```

Reply:
so os_sku and offer are duplicated and os_version and sku_name are duplicated.

Reply:
Remove os_sku and os_version from the JSON and instead we now pass the value of os_sku through offer_name.